packages/ai/cypress/specs/Input.cy.tsx

```diff
@@ -0,0 +1,950 @@
+                    backwards: true
+                });
+        });
+
+        it("should handle Ctrl+Shift+Y for next version when multiple versions exist", () => {
+            const onVersionChange = cy.spy().as("onVersionChange");
+
+            cy.mount(
+                <AIInput
+                    loading={false}
+                    currentVersion={2}
+                    totalVersions={3}
+                    onVersionChange={onVersionChange}
+                />
+            );
+
+            cy.get("[ui5-ai-input]")
+                .shadow()
+                .find("input")
+                .focus()
+                .realPress(['Control', 'Shift', 'Y']);
+
+            cy.get("@onVersionChange")
+                .should("have.been.calledOnce")
+                .its("firstCall.args.0.detail")
+                .should("deep.equal", {
+                    backwards: false
+                });
+        });
+    });
+
+    describe("Input Integration", () => {
+        it("should inherit Input functionality", () => {
+            cy.mount(<AIInput value="Test content" />);
+
+            cy.get("[ui5-ai-input]")
+                .shadow()
+                .find("input")
+                .focus()
+                .should("have.value", "Test content")
+                .type(" additional text");
+
+            cy.get("[ui5-ai-input]")
+                .should("have.prop", "value")
+                .and("include", "Test content")
+                .and("include", "additional text");
+        });
+
+        it("should support readonly mode", () => {
+            cy.mount(<AIInput value="Readonly content" readonly />);
+
+            cy.get("[ui5-ai-input]")
+                .should("have.attr", "readonly");
+
+            cy.get("[ui5-ai-input]")
+                .shadow()
+                .find("input")
+                .should("have.attr", "readonly");
+
+            cy.get("[ui5-ai-input]")
+                .shadow()
+                .find("input")
+                .should("have.value", "Readonly content");
+        });
+
+        it("should support disabled mode", () => {
+            cy.mount(<AIInput value="Disabled content" disabled />);
+
+            cy.get("[ui5-ai-input]")
+                .should("have.attr", "disabled");
+
+            cy.get("[ui5-ai-input]")
+                .shadow()
+                .find("input")
+                .should("have.attr", "disabled");
+
+            cy.get("[ui5-ai-input]")
+                .shadow()
+                .find("input")
+                .should("have.value", "Disabled content");
+        });
+
+        it("should support custom accessible name", () => {
+            cy.mount(<AIInput accessibleName="Custom AI Input" />);
+
+            cy.get("[ui5-ai-input]")
+                .shadow()
+                .find("input")
+                .should("have.attr", "aria-label", "Custom AI Input");
+
+        });
+    });
+
+    describe("Event Handling", () => {

```

Comment:
As above - unless we are not changing anything from the parent component behavior - we should not test it. It is already tested in the parent component